backend/load-test/scripts/collect-results.sh

```
@@ -0,0 +1,204 @@
+date -u +"%Y-%m-%dT%H:%M:%SZ" > "$OUT_DIR/timestamp_utc.txt"
+date +"%Y-%m-%d %H:%M:%S %Z" > "$OUT_DIR/timestamp_local.txt"
+
+# ---------- phase=after 전용 추가 수집 ----------
+
+if [ "$PHASE" = "after" ]; then
+
+  # 8a. 오버셀 검증 (D 필수, A/E도 부수 정보로 확인)
+  echo "[7/8] oversell check"
+  sudo docker exec "$DB_CONTAINER" psql -U postgres -d myfave \
+    -c "SELECT p.product_id, COUNT(*) AS paid_count FROM order_items oi JOIN orders o ON oi.orders_id = o.orders_id JOIN products p ON oi.products_id = p.product_id WHERE o.order_status = 'PAID' AND p.product_id <= 10 AND o.created_at > NOW() - INTERVAL '15 minutes' GROUP BY p.product_id ORDER BY p.product_id;" \
+    > "$OUT_DIR/oversell_check.txt" 2>&1 || true
```

Shellcheck (0.11.0)

[warning] sudo doesn't affect redirects. Use ..| sudo tee file

(SC2024)